proto/check.proto

```diff
@@ -49,11 +49,8 @@ message TDQuoteBodyPolicy {
   bytes mr_config_id = 6;     // should be 48 bytes
   bytes mr_owner = 7;         // should be 48 bytes
   bytes mr_owner_config = 8;  // should be 48 bytes
-  bytes rt_mr0 = 9;           // should be 48 bytes
-  bytes rt_mr1 = 10;          // should be 48 bytes
-  bytes rt_mr2 = 11;          // should be 48 bytes
-  bytes rt_mr3 = 12;          // should be 48 bytes
-  bytes report_data = 13;     // should be 64 bytes
+  repeated bytes rtmr = 9; // should be 48 bytes
```

Comment:
Protobuf style is to use the plural form when using repeated.

Reply:
Done